docs/release-source/release/migrating-to-3.0.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: Migrating to v3.0 - Sinon.JS
+breadcrumb: migrating to 3.0
+---
+
+Sinon v3.0 is the third major release, we have made several breaking changes in this release as a result of modernising the internals of Sinon.  This guide is intended to walk you through the changes.
```

Comment:
This could be more concise:

> As with all `MAJOR` releases in [`semver`](http://semver.org), there are breaking changes in `sinon@3`.
> This guide will walk you through those changes.